frontend/src/metabase/writeback/types.ts

```diff
@@ -60,6 +60,8 @@ export interface HttpActionTemplate {
   parameter_mappings: Record<ParameterId, ParameterTarget>;
 }
 
+export type WritebackRowAction = WritebackActionBase & RowAction;
```

Comment:
Sorry for the confusion, but let's try to call them `QueryActions` instead of `RowActions` in the code for now. Will need to rename what we have now at some point

Reply:
yeah, i much prefer query action too. will refactor this here.